GoogleForCreators/web-stories-wp
# Dashboard: Update list view sort icons

## Summary
Updates list view sorting icons to match designs. Now dates have arrows and titles and authors get a special arrow with a-z or z-a to help indicate the order.

Alpha sort:
![alphademo](https://user-images.githubusercontent.com/10720454/83204220-27525300-a100-11ea-9064-acde8b394d37.gif)

Date sort:
![date demo](https://user-images.githubusercontent.com/10720454/83204246-30432480-a100-11ea-9ba0-8e7dc4338925.gif)


Design video: https://drive.google.com/drive/folders/1onGbNRFVAalJXtbYYFigSbdG-nHUE7As 

## Relevant Technical Choices
- Adds new svg files 
- Adjusts `ArrowIconWithTitle` to give active icon display inline block so that vertical align and height 100% can center the icon to the table header contents. 

## User-facing changes
- New icons on sort of list view in dashboard my stories
- Adds 3 new icons to storybook: 
![Screen Shot 2020-05-27 at 4 11 26 PM](https://user-images.githubusercontent.com/10720454/83081113-c447ba00-a034-11ea-9023-2bc25268cf29.png)


## Testing Instructions
- Make sure that new icons are showing up in dashboard list view
- Make sure new icons are showing up in storybook dashboard/Icons 

---



Fixes #1498
